### PR TITLE
readme creation

### PR DESCRIPTION
**Pull request by 0x2f2f**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,126 @@
-# PADIVAR-Hardware
+# PADIVAR
+
+**PADIVAR** is an open-source OBD2 vehicle diagnostic software compatible with multiple adapters. 
+It is hosted on a server connected to one (or multiple) cars, via one (or multiple) OBD2 devices, and can be accessed simultaneously by multiple clients remotely thanks to a simple TCP client.
```
Comment (poloz-lab) [inline README.md:4]: For now only one client

File: README.md [hunk 2]
```diff
@@ -1 +1,126 @@
...
+   * [License](#license)
+<!--te-->
+
+
+# Installation
+
+## Build
+
+* Download `g++` and `make`
+* Download `libbluetooth-dev` (Debian) or `libbluez3` (Mageia)
```
Comment (poloz-lab) [inline README.md:30]: is it sure for Mageia?

File: README.md [hunk 3]
```diff
@@ -1 +1,126 @@
...
+* Clone project `git clone https://github.com/poloz-lab/PADIVAR-Hardware`
+* Compile with `make`
+
+# Usage
+
+## Setup
+
+In order to work, PADIVAR is separated in two parts:
+* a server (referred here as '*PADIVAR server*') , running either Debian or Mageia (other distributions untested)
+* a TCP client (any simple TCP client supporting CRLF works)
```
Comment (poloz-lab) [inline README.md:40]: only LF

File: README.md [hunk 4]
```diff
@@ -1 +1,126 @@
...
+
+```
+#### Compatible OBD2 devices
+
+PADIVAR can be used with multiple OBD2 adapters, each using different protocols.
+At this time, the following adapters are available:
+* `elm327`
+	 * `bluetooth`
+	 * `wifi`
+	 * `usb`
```
Comment (poloz-lab) [inline README.md:64]: it's still in development

File: README.md [hunk 5]
```diff
@@ -1 +1,126 @@
...
+#### Available commands
+
+At this time, the following commands are available:
+* `quit` - stops the connection
+* `listPID` - list all supported PIDs
+* `description` - displays description of PID
+	* `<PID>` (hex value)
+* `units` - displays the unit of PID value
+	* `<PID>` (hex value)
+
```
Comment (poloz-lab) [inline README.md:76]: is it possible to add the command sendPID (only if a device is connected) that asks for a pid and display the value sent by the car transformed by PADIVAR?

File: README.md (hunk 1 above)
Comment (poloz-lab) [reply README.md:4]: And one adapter

File: README.md (hunk 2 above)
Comment (0x2f2f) [reply README.md:30]: Compiled on Mageia 8 with lib64bluez-devel, works fine.

File: README.md (hunk 5 above)
Comment (poloz-lab) [reply README.md:76]: And repeatPID (only if a device is connected) that asks for a PID and then a time in seconds, and display the value for the time

**Pull request merged**